### PR TITLE
{tools}[GCCcore/10.2.0] iperf v2.1.9

### PR DESCRIPTION
**Pull request by NordMike**

### CODE REVIEW

File: easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'iperf'
+version = '2.1.9'
+
+homepage = 'http://iperf2.sourceforge.net/'
+description = """Iperf-2.1.9: TCP and UDP bandwidth performance measurement tool"""
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/iperf2/files', 'download')]
+checksums = ['5c0771aab00ef14520013aef01675977816e23bb8f5d9fde016f90eb2f1be788']
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb:24]: we should have a `binutils` builddependency when compiling with the `GCCcore` toolchain

style nitpick: lets also move the toolchain paramter up to go between description and sources, for consistency with other easyconfigs

```suggestion
homepage = 'http://iperf2.sourceforge.net/'
description = """Iperf-2.1.9: TCP and UDP bandwidth performance measurement tool"""

toolchain = {'name': 'GCCcore', 'version': '10.2.0'}

sources = [SOURCELOWER_TAR_GZ]
source_urls = [('http://sourceforge.net/projects/iperf2/files', 'download')]
checksums = ['5c0771aab00ef14520013aef01675977816e23bb8f5d9fde016f90eb2f1be788']

builddependencies = [('binutils', '2.35')]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb (hunk 1 above)
Comment (NordMike) [reply easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb]: Thank you for fixes.

File: easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+name = 'iperf'
+version = '2.1.9'
+
+homepage = 'http://iperf2.sourceforge.net/'
+description = """Iperf-2.1.9: TCP and UDP bandwidth performance measurement tool"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (smoors) [inline easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb:23]: the software name is already lower case so we can just use:
```suggestion
sources = [SOURCE_TAR_GZ]
```

File: easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = [('http://sourceforge.net/projects/iperf2/files', 'download')]
```
Comment (smoors) [inline easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb:24]: always use https if possible
```suggestion
source_urls = [('https://sourceforge.net/projects/iperf2/files', 'download')]
```

File: easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = [('https://sourceforge.net/projects/iperf2/files', 'download')]
+checksums = ['5c0771aab00ef14520013aef01675977816e23bb8f5d9fde016f90eb2f1be788']
+
+builddependencies = [('binutils', '2.35')]
+
+sanity_check_paths = {
+    'files': ['bin/iperf'],
+    'dirs': []
+}
+
```
Comment (smoors) [inline easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb:33]: please also add a sanity check command here:
```python
sanity_check_commands = ["iperf --help"]
```

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @NordMike!

**Pull request merged**